website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Amazon Web Services (AWS)"
+linkTitle: "AWS"
+weight: 10
+---
+
+## Control Provisioning with Labels
+
+The [Provisioner CRD]({{< ref "provisioner-crd.md" >}}) supports defining
+instance types, node labels, and node taints.
+
+For certain well-known labels (documented below), Karpenter will provision
+nodes accordingly. For example, in response to a label of
+`topology.kubernetes.io/zone=us-east-1c`, Karpenter will provision nodes in
+that availability zone.
+
+Regarding AWS, 3 types of provisioning constraints are recognized: 
+- [Instance Type](#instance-type-allowlist)

```

Comment:
What about Architecture and Operating System and Zone?

Reply:
Instance type is specified as it's own list in the crd yaml, so I kept it separate. 

"Node labels" are things that go under "labels:" in the provisioner crd.

I also separated the pod GPU stuff because it only makes sense to include a podspec and not the provisioner crd.

I'm very open to alternate ways to structure some page. I did struggle with trying to make it more than a random list.

Reply:
Here's how I think about it:

Vendor neutral constraints (provisioner top level fields, pod.spec.nodeSelector)
- instance type
- zone
- architecture
- operating system

Vendor specific constraints (provisioner.spec.labels, pod.spec.nodeSelector)
- [aws] capacity type
- [aws] launch template
- [aws] security groups
- [aws] subnets